src/utilities/sql/SqlFile_Impl.cpp

```diff
@@ -2800,7 +2800,7 @@ namespace detail {
         // in 8.1 this is 'EnergyPlus-Windows-32 8.1.0.008, YMD=2014.11.08 22:49'
         // in 8.2 this is 'EnergyPlus, Version 8.2.0-8397c2e30b, YMD=2015.01.09 08:37'
         // radiance script is writing 'EnergyPlus, VERSION 8.2, (OpenStudio) YMD=2015.1.9 08:35:36'
-        boost::regex version_regex("\\d\\.\\d[\\.\\d]*");
+        boost::regex version_regex("\\d{1,}\\.\\d[\\.\\d]*");
```

Comment:
@jmarrec Does this look right?

Reply:
Ah yeah right, the version change... I suppose it will always be a two digits year at the start. The minor and patch versions are unlikely to get into the two digits though, but I'm not sure.
 Pinging @Myoldmopar here... Could you let us know a bit more about the new versionning scheme for E+?
 
 Side note: this may be a good opportunity to change this to a raw string litteral to avoid escaping like crazy, like so 

```c++
boost::regex version_regex(R"(\d{2}\.\d[\.\d]*)");
```